README.md

```diff
@@ -50,6 +45,13 @@ Open up your browser to http://localhost:1313 to view the website. As you make c
 
 Make sure to install the Hugo extended version specified by the `HUGO_VERSION` environment variable in the [`netlify.toml`](netlify.toml#L10) file.
 
+Install dependencies:
+
+```
+sudo npm install -D --save autoprefixer
+sudo npm install -D --save postcss-cli
```

Comment:
This doesn't look right - we don't want to insist that contributors run commands as the superuser if we can avoid it.

Reply:
Can we skip the `sudo`?

Reply:
OK. I pulled the commands from the theme's README.